.github/workflows/continuous-integration.yaml

```diff
@@ -402,10 +402,10 @@ jobs:
     strategy:
       matrix:
         os: [ linux ]
-        target: [ cardano-node-ogmios ]
+        target: [ ogmios ]
```

Comment:
Hmm, that's not going to work; the cardano-node is specifically needed for the `cardano-node-ogmios` target for the image.

Reply:
sorry i forked the repo for creating image. Didn't realised i created a PR here :D

Reply:
Well, I was assuming you might do one. I would happily merge it :)

Reply:
I will do one in that case. Better for me as well.

Reply:
Are you working on it already @KtorZ ? I see some recent changes related to v10.1.3

Reply:
Indeed. Thought I might as well just do it given it's a few lines.